mangle-shebangs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SYSTEM_RELEASE_PATH=/etc/system-release
+
+# RHEL8 and Fedora30 both treat shebangs of the form #!/usr/bin/env
+# python as errors, and both default to python2
+if [ -f $SYSTEM_RELEASE_PATH ] && [[ "$(cat $SYSTEM_RELEASE_PATH)" =~ Red\ Hat\ Enterprise\ Linux\ release\ 8|Fedora\ release\ 30 ]]; then
```

Comment:
What about
```bash
RHEL8_REGEX="Red Hat Enterprise Linux release 8"
FEDORA_REGEX="Fedora release"
if [ -f $SYSTEM_RELEASE_PATH ] && [[ "$(cat $SYSTEM_RELEASE_PATH)" =~ $RHEL8_REGEX|$FEDORA_REGEX ]]; then
    # Replace the first line in .py to "#!/usr/bin/env python2" no matter what it was before
    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python2/' src/watchdog/__init__.py
    sed -i -e '1 s/^.*$/\#!\/usr\/bin\/env python2/' src/mount_efs/__init__.py
fi
```
This is much easier to introduce other distribution.

Reply:
I made these changes in commit 7f4d4c6.